RocketApp/app/src/main/java/com/example/rocketapp/BinomialExperiment.java

```diff
@@ -0,0 +1,67 @@
+package com.example.rocketapp;
+import java.util.ArrayList;
+import java.lang.Integer;
+import com.google.firebase.firestore.Exclude;
+
+public class BinomialExperiment extends Experiment {
+    public static String TYPE = "Binomial";
+
+    private ArrayList<BinomialTrial> trials = new ArrayList<>();
```

Comment:
should be able to remove this variable and replace any uses of it in this class to getTrials()